src/PageLibrary/RegistrationPage.java

```diff
@@ -0,0 +1,5 @@
+package PageLibrary;
+
+public class RegistrationPage {
```

Comment:
i did not code please add some more code so that it will help the project

Reply:
so what approve it